src/Evolution/Systems/Ccz4/TempTags.hpp

```diff
@@ -34,5 +34,185 @@ template <size_t Dim, typename Frame, typename DataType>
 struct GammaHatMinusContractedConformalChristoffel : db::SimpleTag {
   using type = tnsr::I<DataType, Dim, Frame>;
 };
+
+/*!
+ * \brief The CCZ4 temporary expression \f$K - 2 \Theta c\f$
+ *
+ * \details Here, \f$K\f$ is the trace of the extrinsic curvature defined by
+ * `gr::Tags::TraceExtrinsicCurvature`, \f$\Theta\f$ is the projection of the Z4
+ * four-vector along the normal direction, and \f$c\f$ is ???.
```

Comment:
@nilsdeppe I wasn't sure how to describe what exactly c is in words. Let me know how you think this should be described!